libheif/box.cc

```diff
@@ -1003,12 +1013,14 @@ Error Box_iloc::parse(BitstreamRange& range)
     for (int e = 0; e < extent_count; e++) {
       Extent extent;
 
-      if (index_size == 4) {
-        extent.index = range.read32();
-      }
-      else if (index_size == 8) {
-        extent.index = ((uint64_t) range.read32()) << 32;
-        extent.index |= range.read32();
+      if (get_version() >= 1 && index_size > 0) {
```

Comment:
I guess that this is accidentally reverting commit a11e10cab275097971089b227e23f8dcc5ca3f93.
Please remove this from the PR or explain why you reverted the above commit.

Reply:
Indeed accidental. My bad.